app/views/feed/_feed_items.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="items mt-0">
+  <% if @revisions.any? %>
+    <% @revisions.each do |revision| %>
+      <% if !@marker_shown && @feed_last_visited_at && revision.created_at < @feed_last_visited_at %>
+        <% @marker_shown = true %>
+
+        <h2 class="feed__heading">Already seen updates</h2>
+      <% end %>
+
+      <%= render "feed_item", revision: revision %>
+    <% end %>
+
+    <% if @revisions.count == 20 %>
+      <% unless current_user && current_user.pagination_type != "infinite_scroll" %>
+        <div class="infinite-scroll" data-role="infinite-scroll-marker" data-url="<%= request.original_url %>"></div>
+      <% end %>
+
+      <% if current_user && current_user.pagination_type == "load_more" %>
+        <div data-role="load-more-posts-marker"></div>
+        <div class="flex justify-center">
+          <div class="mt-1/2 button button--secondary pr-1/1 pl-1/1" data-role="load-more-posts" data-url="<%= request.original_url %>">Load more</div>
```

Comment:
```suggestion
          <div class="mt-1/2 button button--secondary pr-1/1 pl-1/1" data-role="load-more-posts" data-url="<%= request.original_url %>" data-load-method="load-more-button">Load more</div>
```
As per changes made in https://github.com/EloHellEsports/workshop.codes/pull/146/commits/cb745133be95a38c7e22c1dd27fe53adf3a7269a, the infinite scroll module now only has defined behavior for elements with an explicit `data-load-method`.